manifests/components/kibana.jsonnet

```diff
@@ -38,18 +39,69 @@ local strip_trailing_slash(s) = (
     },
   },
 
+  // List of Kibana plug-ins to install
+  plugins:: {
+    /* Example:
+    logtrail: {
+      version: "v0.1.31",
+      url: "https://github.com/sivasamyk/logtrail/releases/download/v0.1.31/logtrail-6.7.1-0.1.31.zip",
+    },
+    */
+  },
+
   es: error "elasticsearch is required",
 
   serviceAccount: kube.ServiceAccount($.p + "kibana") + $.metadata {
   },
 
+  pvc: kube.PersistentVolumeClaim($.p + "kibana-plugins") + $.metadata {
+    storage: "15Gi",
```

Comment:
Isn't is this too much space for the plugins?

Also, if by default we are not going to install any plugin we could have less space assigned to that volume

Reply:
Good point. I've reduced that to 2.